src/components/notifications/AccountNotifications.tsx

```diff
@@ -154,15 +178,65 @@ export const AccountNotifications: FC<IAccountNotifications> = (
                 shape="circle"
                 spacing="compact"
                 appearance="subtle"
-                onClick={(event: MouseEvent<HTMLElement>) => {
-                  // Don't trigger onClick of parent element.
-                  event.stopPropagation();
-                  markNotificationsRead(notifications);
-                }}
+                onClick={openModal}
```

Comment:
does open item 1 get resolved when you do
```
onClick={(event: MouseEvent<HTMLElement>) => {
  // Don't trigger onClick of parent element.
  event.stopPropagation();
  openModal();
}}
```

Reply:
Nope :( 
Added that regardless.

Reply:
It's because of
```
<Box onClick={toggleAccountNotifications}
```

Have a few ideas to fix it (adding stopPropagation on the isOpen/isClosed functions..)

Reply:
Fixed it! Moving the Modal outside of the `Box` did the trick